ExposureTests/EnvironmentSpec.swift

```diff
@@ -17,10 +17,12 @@ class EnvironmentSpec: QuickSpec {
             let base = "http://base.url.com"
             let customer = "TestCustomer"
             let businessUnit = "TestBusinessUnit"
-            let basePath = "/v1/customer/" + customer + "/businessunit/" + businessUnit
-            let env = Environment(baseUrl: base, customer: customer, businessUnit: businessUnit)
+            let version = "v1"
+            let basePath = "/" + version + "/customer/" + customer + "/businessunit/" + businessUnit
             
```

Comment:
It would be good practice to add a unit test for inequality between different versions too

Reply:
I guess it will be to check the inequality between env.version & the version. Something like 
`it ("should equal the versions") {
                expect(env.version).to(equal(version))
            }`